packages/office-ui-fabric-react/src/components/Persona/PersonaPresence/PersonaPresence.base.tsx

```diff
@@ -72,30 +74,27 @@ export class PersonaPresenceBase extends BaseComponent<IPersonaPresenceProps, {}
   }
 
   private _onRenderIcon = (className?: string, style?: React.CSSProperties): JSX.Element => (
-    <Icon className={className} iconName={this._determineIcon()} style={style} />
+    <Icon className={className} iconName={determineIcon(this.props.presence, this.props.isOutOfOffice)} style={style} />
   );
+}
 
-  private _determineIcon = (): string | undefined => {
-    const { presence } = this.props;
+function determineIcon(presence: PersonaPresenceEnum | undefined, isOutOfOffice: boolean | undefined): string | undefined {
+  if (!presence) {
+    return undefined;
+  }
 
-    if (presence !== PersonaPresenceEnum.none) {
-      let userPresence = PersonaPresenceEnum[presence as PersonaPresenceEnum];
+  const oofIcon = 'SkypeArrow';
 
-      switch (userPresence) {
-        case 'online':
-          userPresence = 'SkypeCheck';
-          break;
-        case 'away':
-          userPresence = 'SkypeClock';
-          break;
-        case 'dnd':
-          userPresence = 'SkypeMinus';
-          break;
-        default:
-          userPresence = '';
-      }
+  switch (PersonaPresenceEnum[presence]) {
+    case 'online':
+      return 'SkypeCheck';
+    case 'away':
+      return isOutOfOffice ? oofIcon : 'SkypeClock';
+    case 'dnd':
+      return 'SkypeMinus';
+    case 'offline':
+      return isOutOfOffice ? oofIcon : '';
```

Comment:
Hey @Markionium in the screenshot attached, I see that we have a cross icon in case of `offline`. Did we miss adding that here?

Reply:
We didn't miss adding it here. It should be added still but it is not available in the current icon set. @mattcoxonline is looking into how we can get the icon added to the set.

Reply:
Ah, okay. 👍